### ISSUE TITLE
Model for ATS

### ISSUE THREAD

**Issue by scorpio1441** [umutkcbs/hope-autonomous-driving#4]
Hello, do you have a model for American Truck Simulator?

I'm having hard times running on 1.47.1
I know you mentioned about the mouse steering, could you please send me the updated code with virtual controller?

Thanks.

**Comment by umutkcbs** [comment]
Hi, 
I did not try the existing model in ATS. However, it should perform nearly the same as in ETS with proper FOV and lighting tuning.
It looks like SCS doesn't want us to run this thing in their games (because they instantly changed one of the core mechanics in the engine). **So** I do not want to send the exact virtual controller code, but I will show you how you can update your code accordingly.
Please understand my excuse; I will do this as soon as I go back home.

**Comment by scorpio1441** [comment]
@umutkcbs If this implies using vjoy or xbox virtual controllers, I have figured this out. =)

**Comment by umutkcbs** [comment]
Yes, I used vgamepad and ds4 but if it is working all of them ok. I came back home, in few minutes I will commit a file which I used to test virtual controllers in different games.
Seeing other people trying to work on this project made me very happy, so I want to thank you so much.

I will keep open this thread while you working on. Do not hesitate to ask me.
Have a good day

**Comment by scorpio1441** [comment]
@umutkcbs I've tested pretty much all the related projects on GitHub and contacted multiple developers. Spent weeks on researches. The conclusion is - the lane recognition is not reliable at all.

I'm currently working on getting the lane/truck positions directly from memory. Have you seen the [SCS Blender Tools](https://modding.scssoft.com/wiki/Documentation/Tools/SCS_Blender_Tools)? I may provide some clues on how it works.

**Comment by umutkcbs** [comment]
Since game graphics are improving, lane detection is getting harder and harder like real-life scenarios. If your aim is 'only driving to a given location,' adjusting graphical buffer settings in the console might help you.
![image_2023-07-03_135233455](https://github.com/umutkcbs/hope-autonomous-driving/assets/49411067/c14a29b9-42cf-4169-a2ff-28ca3c3fad0f)

Also [SCS SDK plugin](https://github.com/RenCloud/scs-sdk-plugin) and [ETS2 Telemety](https://github.com/Funbit/ets2-telemetry-server) gives nearly every info about game real-time and you can get values for your script. Maybe they can give you the positions you looking for
